platform/ios/src/MGLMapView.mm

```diff
@@ -4312,15 +4312,15 @@ - (void)validateLocationServices
         if ([CLLocationManager authorizationStatus] == kCLAuthorizationStatusNotDetermined)
         {
             BOOL requiresWhenInUseUsageDescription = [NSProcessInfo.processInfo isOperatingSystemAtLeastVersion:(NSOperatingSystemVersion){11,0,0}];
-            BOOL hasWhenInUseUsageDescription = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSLocationWhenInUseUsageDescription"];
+            BOOL hasWhenInUseUsageDescription = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSLocationWhenInUseUsageDescription"] ? YES : NO;
```

Comment:
I’m fond of using the ‼️ operator for this, but writing it out explicitly is fine.

Reply:
Actually, if you want to be explicit, I’d recommend comparing to `nil`.

Reply:
Went with `!!`... for _style_.